web/templates/settings/profile_settings.hbs

```diff
@@ -63,7 +63,7 @@
             </div>
             <button class="button rounded sea-green w-200 block" id="show_my_user_profile_modal">
                 {{t 'Preview profile' }}
-                <i class="fa fa-external-link" aria-hidden="true" title="{{t 'Preview profile' }}"></i>
+                <i class="fa fa-external-link"></i>
```

Comment:
Hmm, should this have `aria-hidden` on it still? I would expect so giving it's not an interactable or readable element, just a visual decoration.